packages/core/src/components/select-item/select-item.tsx

```diff
@@ -55,12 +58,14 @@ export class SelectItem implements DropdownItemWrapper {
   /**
    * Item clicked
    */
-  @Event() itemClick: EventEmitter<string>;
+  @Event() itemClick!: EventEmitter<string>;
+
+  private componentLoaded = false;
 
   /** @internal */
   @Method()
   async getDropdownItemElement(): Promise<HTMLIxDropdownItemElement> {
-    return this.dropdownItem;
+    return this.dropdownItem as HTMLIxDropdownItemElement;
```

Comment:
Possibly better solution?
```suggestion
    return this.dropdownItem!;
```